src/modules/heritability_proxy/build-and-push-starter.sh

```diff
@@ -15,7 +15,7 @@ set +a
 source ../../../env/${ENV}/global.env
 
 # this requires reading the respective global.env above
-IMAGE_URI=gcr.io/${GOOGLE_CLOUD_PROJECT_ID}/heritability:v0.02-debug
+IMAGE_URI=gcr.io/${GOOGLE_CLOUD_PROJECT_ID}/heritability:v0.04
```

Comment:
is there a way to avoid hardcoding version information in this file and instead load it from the global.env?